kpi/migrations/0026_disable_editors_can_change_permissions.py

```diff
@@ -0,0 +1,39 @@
+import sys
+
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    sys.stderr.write(
+        'Disabling `editors_can_change_permissions` on all assets and '
+        'collections...'
+    )
+    sys.stderr.flush()
+
+    Asset = apps.get_model('kpi', 'Asset')  # noqa
+    Collection = apps.get_model('kpi', 'Collection')  # noqa
+    Asset.objects.update(editors_can_change_permissions=False)
+    Collection.objects.update(editors_can_change_permissions=False)
+
+
+def reverse_func(apps, schema_editor):
+    sys.stderr.write(
+        'ENABLING `editors_can_change_permissions` on all assets and '
```

Comment:
Did you put this in CAPS in purpose? (vs `Disabling` in the `forward_func`)

Reply:
Yes, exactly :)